packages/create-turbo/src/commands/create/updateCommandsInREADME.js

```diff
@@ -0,0 +1,29 @@
+const fs = require("fs").promises;
```

Comment:
This needs to be written in Typescript and not CommonJS

Reply:
Done